packages/transport/src/modules/relationships/RelationshipsController.ts

```diff
@@ -191,32 +192,63 @@ export class RelationshipsController extends TransportController {
         if (!relationship) {
             throw CoreErrors.general.recordNotFound("Relationship", relationshipId.toString());
         }
+        if (!relationship.cache) {
+            await this.updateCacheOfRelationship(relationship);
+        }
+        if (!relationship.cache) {
+            throw this.newCacheEmptyError(Relationship, relationshipId.toString());
```

Comment:
Probably out of context, but I wonder why this error is defined in the TransportController and not the CoreErrors file of the transport lib, where all other errors are defined (with error codes).

Reply:
This error is not supposed to happen in production, just in development

Reply:
@Milena-Czierlinski CoreErrors (also all errors with error codes) are errors that the user of the library cares about. This error indicates a bad state of the library and the user will have no possibility to do anything about it.